scripts/qa_crowbarsetup.sh

```diff
@@ -3478,10 +3478,14 @@ function oncontroller_testsetup()
     fi
 
     # wait for nova-manage to be successful
-    for n in $(seq 1 200) ;  do
-        test "$(nova-manage service list  | fgrep -cv -- \:\-\))" -lt 2 && break
-        sleep 1
-    done
+    if iscloudver 7plus; then
+        wait_for 200 1 "test \"$(nova service list  | fgrep -cv -- \ up\ )\" -lt 5" "Nova services to be up and running"
+    else
+        for n in $(seq 1 200) ;  do
+            test "$(nova-manage service list  | fgrep -cv -- \:\-\))" -lt 2 && break
```

Comment:
sure, just didn't want to break the old case (or rather, spend the testing time on it)